### ISSUE TITLE
[Feature Request] Option to return Picked Media's original Album Path

### ISSUE THREAD

**Issue by Jazzeroki** [jamesmontemagno/Xamarin.Plugins#164]
Just tested so far with Android but the Path extension is returning a path inside of a temp directory.  If I use AlbumPath when taking a photo that is working great.  My trouble currently is when I pick a photo that photo path isn't usable to pass to another application like say an image cropper or exif reader.  Album path parameter be changed to also work when picking a photo not just saving one.  Might also be good to do with video I just haven't needed that yet.

**Comment by jamesmontemagno** [comment]
Ah, I see, yeah let me see what I can do as we copy a version locally that can be modified as you probably shouldn't actually touch the original.

What would need to happen is on the selection a copy needs to be created and also if you want to save to album copy it to album. Interesting.

**Comment by Jazzeroki** [comment]
Let me add this issue relates back to 156.  I didn't say anything about iOS because I hadn't tested it yet at the time.  Not having a value in Album Path for iOS was preventing me from getting some needed Exiff info.  I've attached a file I worked with another developer on to get the album path in there for iOS.  
[MediaPickerDelegate.cs.txt](https://github.com/jamesmontemagno/Xamarin.Plugins/files/101108/MediaPickerDelegate.cs.txt)
